definitions/divorce/json/CaseEvent.json

```diff
@@ -692,6 +692,8 @@
     "DisplayOrder": 1,
     "PreConditionState(s)": "AwaitingPronouncement",
     "PostConditionState": "AwaitingDecreeAbsolute",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/generate-dn-pronouncement-documents",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```

Comment:
what does `120, 120` mean? try twice every 2 minutes?

Reply:
Yeah, its retry if failed after 2 mins.
Just for consistency, I think we need to make a proper decision on our retries, but as far as I'm aware its not even working right now.
This CCD ticket is to implement it:
https://tools.hmcts.net/jira/browse/RDM-4522
Dunno if having 2 mins is too long.